pyincore_viz/geoutil.py

```diff
@@ -1140,6 +1140,99 @@ def get_ipyleaflet_heatmap(locations=None, radius=10, blur=10, max=1, name=""):
 
         return heatmap
 
+    @staticmethod
+    def plot_local_earthquake(eq_dataset):
+        """
+        Plot local earthquake data on the map
+        """
+        demand_type = eq_dataset.demand_type
+        demand_units = eq_dataset.demand_units
+        hazard_type = eq_dataset.hazard_type
+        period = eq_dataset.period
+        title = "Demand Type: " + demand_type.upper() + ", Demand Units: " + demand_units + ", Period: " + \
```

Comment:
Just an aesthetic suggestion, not necessary... but we can use .title() instead of .upper() for the title so the resulting title is not all in uppercase?